packages/@okta/vuepress-site/code/angular/okta_angular_auth_js/index.md

```diff
@@ -263,11 +255,19 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-secure',
-  template: ``
+  templateUrl: './protected.component.html',
 })
-export class ProtectedComponent {
-  constructor() { console.log('Protected endpont!'); }
-}
+export class ProtectedComponent {}
+```
+
+Create a new template called `protected.component.html`:
+
+```html
+<!-- protected.component.html -->
+
+<h2>PROTECTED!</h2>
+
+<router-outlet></router-outlet>
```

Comment:
I don't think router-outlet is needed here.

Reply:
fixed...